### ISSUE TITLE
The Tab component lacks a11y

### ISSUE THREAD

**Issue by pettersmoen** [dhis2/ui-core#560]
The tab component has been given a style of `outline: none`, without any other alternative focus styling, which makes it really painful to navigate without a mouse. You should consider adding a outlining to it, as you have done with other components, such as the Button.

**Comment by HendrikThePendric** [comment]
Thanks for reporting. @cooper-joe do you have any suggestions with regards to this? Perhaps activating the hover styles on `focus` as well? And making sure this gets overridden by the `selected` style? IMO an outline wouldn't look very good in a TabBar....

**Comment by cooper-joe** [comment]
Yes, we should definitely activate the hover styles on `focus `, good idea. Although I don't think that just the hover style is clear enough for keyboard navigation.

Is it possible to, when the tab has focus, outline the text only? 

Here is a rough illustration of what I mean:
![image](https://user-images.githubusercontent.com/33054985/68844323-e5b10c00-06c9-11ea-9146-872644e9564e.png)

**Comment by HendrikThePendric** [comment]
Great idea! I'll work on this once we get `next` released 👍

**Comment by HendrikThePendric** [comment]
@cooper-joe I've given this a little try and it's definitely possible. Could you let me know:
- padding around the text
- border color/thickness/style

**Comment by cooper-joe** [comment]
Thanks @HendrikThePendric. Let's go for:
```
border: 1px dotted var(--grey900);
padding: 0px 2px;
```
(based on an application directly on the text)

**Comment by cooper-joe** [comment]
Still relevant. @HendrikThePendric I just tested the latest ui-core and these styles weren't present. Do you think we can add them now, or is it better to wait until `ui` 5?

**Comment by HendrikThePendric** [comment]
@cooper-joe this is not a breaking change, so there's nothing really preventing us from doing this now. However, since we are migrating this repo to `@dhis2/ui`, I think it'd be better to work on it there.
